charts/radar-jdbc-connector/templates/configmap.yaml

```diff
@@ -28,3 +28,6 @@ data:
     max.retries = 30
     # Wait 15 minutes before retrying failed task
     retry.backoff.ms = 900000
+    connection.attempts = 20
```

Comment:
Just to check: this will leave the container in an unconnected state for quite long (5 hours) before the connection actually fails. Two questions:
- If the connection finally stops, does the component report become unhealthy so the K8s controller can restart it?
- Why retry 20 times and not let the K8s controller restart the component, just in case something funny is going on with the node or the pod's connections?

Reply:
> If the connection finally stops, does the component report become unhealthy so the K8s controller can restart it?

For some reason, it doesn't restart it automatically. It does become unhealthy because the tasks are killed. I am still looking into it, and figured this could be a temporary solution in the meantime?

> Why retry 20 times and not let the K8s controller restart the component, just in case something funny is going on with the node or the pod's connections?

Because I noticed the timescaledb pod goes down quite often (around 2-3 times a month), but immediately restarts a few minutes after. I can lower the attempts and backoff ms.